src/data/GeboorteplaatsData.ts

```diff
@@ -0,0 +1,11 @@
+import { FormFieldDefinition } from '../form/state/FormFieldDefinition.model';
+import { teletex } from './TeletexData';
+
+export const placeOfBirthValidation: FormFieldDefinition = {
+  maxLength: 40,
```

Comment:
Hmmm, this is probably set by some registry? People born in the town with the longest name will now not be able to fill in our form 😅 https://en.wikipedia.org/wiki/Taumatawhakatangi%C2%ADhangakoauauotamatea%C2%ADturipukakapikimaunga%C2%ADhoronukupokaiwhen%C2%ADuakitanatahu

Reply:
I moved everything into a `nl-NL/` directory to make clear these are not intended as international defaults.